app/views/external_users/claims/case_details/_fields.html.haml

```diff
@@ -22,6 +25,12 @@
           = f.collection_select :external_user_id, @provider.advocates, :id, :name_and_number, { include_blank: '&#160;'.html_safe }, {class: 'form-control autocomplete'}
           = validation_error_message(@error_presenter, :external_user)
 
+        .form-col
+          = f.adp_text_field :providers_ref, label: t('.providers_ref'), errors: @error_presenter
+
+    - else
+      = f.adp_text_field :providers_ref, label: t('.providers_ref'), errors: @error_presenter
```

Comment:
I think we might need a .form-col here as it looks weird and too big otherwise and probably we can make it so there is no need to duplicate this line of code.

<img width="571" alt="screen shot 2016-09-15 at 13 40 41" src="https://cloud.githubusercontent.com/assets/687910/18550112/358e5584-7b4a-11e6-8419-98ee47cf9872.png">